services/github/src/handlers/utils.js

```diff
@@ -193,6 +193,32 @@ export const isCollaborator = async (instGithub, organization, repository, login
   }));
 };
 
+export const getTimeDifference = (timestamp1, timestamp2) => {
+
```

Comment:
probably would be good to check if timestamps and dates that come here are valid. From the code that uses it it might seem as those values might be `undefined` so some check and tests would be helpful

Reply:
For writing the tests, I am first facing issue in running the tests, will get back to you in case I am stuck for too long.

Reply:
any luck with the tests and checking if passed timestamps are defined?

Reply:
I have written the test for this function, will need to add a undefined check for this, slipped out of my mind while updating tests for existing statusHandler function

Reply:
Have written tests for this function, need to add a undefined check, slipped out of my mind while writing tests for existing statusHandler function

Reply:
Also if the timestamps are undefined, we will ideally want to remove it from the check run output right

Reply:
I think it is highly unlikely we won't have any values in there, but if something is missing we can replace value with `n/a`. Would be easier to spot that something is wrong, as if we would hide the line, it wouldn't be visible

Reply:
added n/a condition in check run output for invalid dates.

Reply:
thanks!